### DISCUSSION TITLE
Pertubation Simulations tutorial

### DISCUSSION THREAD

**Discussion by laki-spk** [aertslab/scenicplus#365] [General]
Hello Team,

I was working on the perturbation simulations tutorial and noticed today that it has suddenly been removed from the page. @SeppeDeWinter and the team, would it be possible to bring it back? 
Also, would it be possible to use that feature in future? Part of my research project is based on perturbation simulations. 

Thank you.

**Comment by SeppeDeWinter** [comment]
Hi @laki-spk 

It's still possible to use the perturbation simulation, no worries! 
It has been removed because the tutorials have not been updated yet. However, you can still find the notebook here: https://github.com/aertslab/scenicplus/blob/old/notebooks/Perturbation_simulation.ipynb.

I hope that helps?

All the best,

Seppe
